1001-2000/LIT1693John.xml

```diff
@@ -81,34 +83,61 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                      <citedRange unit="page">18</citedRange>
                   </bibl>
                </listBibl>
-            </div>
 
-            <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="TituliJohn">
-               <label>Tituli of the Gospel of John</label>
-            </div>
 
             <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="IntroJohn">
-               <label>Introduction to the Gospel of John</label>
-            </div>
-
-            <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="BioJohn">
+            <label>Preface to the Gospel of John</label>
+            <div type="textpart" subtype="title expanded" xml:id="TitlePrefaceJohn"><ab>በስመ፡
+             እግዚአብሔር፡ ዘይትዋሓድ፡ በመለኮት፡ ወይሤለስ፡ በኣካላት፡ መቅድመ፡ ወንጌሉ፡ ለዮሐንስ፡ ፈጻሜ፡ ብስራት።</ab></div>
+            <div type="textpart" subtype="subtextunit" xml:lang="gez" xml:id="BiographyJohn">
                <label>Biography of John</label>
+               <div type="textpart" subtype="incipit"><ab>ዝንቱሰ፡ ዮሐንስ፡ ፩፡ እም፲ወ፪፡ ሓዋርያት። ስማ፡ ለእሙ፡
+               ባውፍልያ፡ ወካዕበ፡ ተሰምየት፡ ማርያም። ወውእቱ፡ እምነገደ፡ ዛብሎን፡ ወእምነ፡ ቤተ፡ ሳይዳ፡ ጸሓፈ፡ ወንጌሎ፡ በልሳነ፡ ዮናናውያን፡
+               በ፰፡ ዓመት፡ እመንግሥተ፡ ኔሮን፡ ወልደ፡ አቅሎንድዮስ፡ ዘቀተሎሙ፡ ለጴጥሮስ፡ ወለጳውሎስ። በ፲ወ፫፡ ዓመተ፡ መንግሥቱ፡ እምድኅረ፡
+               ዕርገቱ፡ ለእግዚእነ፡ በ፴፡ ዓመት።</ab></div>
+               <div type="textpart" subtype="explicit"><ab>ወተብህለ፡ ከመ፡ ዮሐንስ፡ ኣወፈዮ፡ መጽሓፈ፡ አቡቀለምሲስ፡ ለፋጊር።</ab></div>
+            </div>
+            <div type="textpart" subtype="subtextunit" xml:lang="gez" xml:id="EusebianSectionsJohn">
+              <label>Statement about Eusebian Sections in the Gospel of John</label>
+              <div type="textpart" subtype="incipit"><ab>ወኍልቈ፡ ምዕራፈ፡ ወንጌሉ፡ ንኡሳን። ፪፻፡ ፴ወ፪፡</ab></div>
+              <div type="textpart" subtype="explicit"><ab>ወምዕራፋቲሁ፡ በልሳነ፡ ሮሜ። ፳</ab></div>
+            </div>
+            <div type="textpart" subtype="subtextunit" xml:lang="gez" xml:id="TituliJohn">
+              <label>Tituli of the Gospel of John</label>
+              <div type="textpart" subtype="incipit"><ab>፩፡ ተአምር፡ ዘቃና፡ ዘገሊላ። ፪፡ ዘከመ፡ ሰደደ፡ እምኰራብ። ፫፡ ኒቆዲሞስ።</ab></div>
+              <div type="textpart" subtype="explicit"><ab>፲፰፡ ዘከመ፡ ነሥኣ፡ ዮሴፍ፡ ሥጋሁ። ፲፱፡ ዘከመ፡ ዜነዉ፡ መላእክት፡ ትንሣኤ፡ ለአንስት።
+              ፳፡ በዊኦቱ፡ ኀበ፡ አርድእት፡ እንዘ፡ ዕፅው፡ ኃዋኅው።</ab></div>
+            </div>
             </div>
 
             <div type="textpart" subtype="chapter" xml:lang="gez" corresp="LIT2715John">
                <label>Bǝsrāta Yoḥannǝs</label>
+               <div type="textpart" subtype="subtextunit" xml:lang="gez" xml:id="JohnGospelHeadline">
+               <label>Headline of the Gospel of John</label>
+               <ab><seg type="title inscriptio expanded">ብስራተ፡ አብ፡ ቅዱስ፡ ትሩፍ፡ ረድእ፡ ተናገሪ፡ በመለኮት፡ ዮሓንስ፡ ወልደ፡
+                ዘብዴዎስ፡ ፍቁረ፡ እግዚእነ፡ ኢየሱስ፡ ክርስቶስ።</seg> ጸሎቱ፡ ወበረከቱ፡ የሀሉ፡ ምስለ፡ ገብሩ፡ ዘመለኮት፡ ለዓመለ፡ ዓለም፡ አሜን፡</ab>
```

Comment:
Again, I would replace ዘመለኮት፡ with `<gap reason="ellipsis" resp="Abr"/>`

Reply:
Ok, Massimo. Thank you! I will do the same everywhere in the Works when I have a proper name